packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -174,7 +174,7 @@ const OpenQueueCard = ({
                 size={96}
                 photoURL={staffMember.photoURL}
                 name={staffMember.name}
-                style={{ marginRight: "25px" }}
+                style={{ marginRight: "25px", marginTop: "10px" }}
```

Comment:
nitpicky especially since we're also not very self-consistent on these, but could you turn these into styled components?

Reply:
do you mean something like this? 
![image](https://user-images.githubusercontent.com/18249360/109117221-8f1ce500-76f6-11eb-838a-0db3b250b4c1.png)

Reply:
yeah

Reply:
is the idea to wrap the existing component,KOHAvatar,in this new styled component?

Reply:
is the idea to wrap the existing component, KOHAvatar, with the styled component?

Reply:
Yup! That'd be ideal.

Reply:
Sorry this is taking so long. Wrapping it in a div causes the profile circles to be in one large column. I've tried wrapping the `<Tooltip>` tag as well as the outer `<div>` that wraps the staffList.map with the styled component, too, but it seems that they lose the ability to be individual components that can be put in a row, and instead be one larger group that makes them become a column. Photo below: 
![Screen Shot 2021-02-26 at 1 20 36 PM](https://user-images.githubusercontent.com/18249360/111039764-932e4f80-83e4-11eb-8eb4-5e7333ad8e14.png)
Maybe there are others ways to extract the styling out besides a "styled component"?

Reply:
My css knowledge is pretty limited so unfortunately I'm not sure why that's the case.

@liustanley or @IrisLiu-00 got any ideas on this?

Reply:
I'm guessing it's because divs are block display by default, though don't quote me on that. You could try setting the div that's holding all the avatars to `display: flex`, then messing with the other flex settings to get the avatars to wrap lines. 

What might be easier is to convert `KOHAvatar` itself into a styled component - so you would do something like 
```
const StyledKOHAvatar = styled(KOHAvatar)` ... put whatever css attributes here...`
```
and then you can just directly use `StyledKOHAvatar` without having to introduce an extra div.

Reply:
Hey Iris, thanks for the ideas! I tried the latter, which for some reason didn't impact the css. The CSS in the styled component didn't seem to be reflected in the app, which I found strange since I used the same structure as other styled components (sent an email with additional info). I'll keep messing around with flex.

Reply:
Thanks to @IrisLiu-00 I was able to get the styled component to work. I have pushed another commit